### PR TITLE
GROUP_CONCAT function support

### PR DESCRIPTION
**Pull request by sdcooke**

### CODE REVIEW

File: tinyquery/evaluator_test.py [hunk 1]
```diff
@@ -945,6 +945,32 @@ def test_null_count_distinct(self):
             'SELECT COUNT(DISTINCT val1) FROM some_nulls_table',
             self.make_context([('f0_', tq_types.INT, [2])]))
 
+    def test_group_concat(self):
+        self.assert_query_result(
+            'SELECT GROUP_CONCAT(str) FROM string_table',
+            self.make_context([
+                ('f0_', tq_types.STRING, ['hello,world'])
+            ]))
+        self.assert_query_result(
```
Comment (csilvers) [inline tinyquery/evaluator_test.py]: Is this the case that's testing a repeated field?

[ I apologize for the silly quesstion; I'm not actually that familiar with tinyquery, but the author has left Khan so I'm the best that's left. :-) ]

**Comment by sdcooke** [comment]
Good spot - I've renamed to GROUP_CONCAT_UNQUOTED (which is what we wanted to use anyway).

That is a test on a repeated field. I'm not actually sure if GROUP_CONCAT_UNQUOTED supports repeated fields and I don't have any repeated fields in bigquery I can test with. Should be easy to remove that functionality and test if bigquery doesn't support it.

**Comment by csilvers** [comment]
Gotcha -- so you're saying your code supports repeated fields but you're not confident that matches bigquery's behavior.  If so, can you add a TODO to check that?  That way if bigquery ends up *not* supporting that, and someone comes along wondering why we do, it will be obvious.

Once the TODO is in, this is good to merge by me.

**Comment by sdcooke** [comment]
TODO added - thanks!

**Comment by csilvers** [comment]
Drat, I forgot to change the commit message before landing. :-/  We'll just have to remember this is actually for GROUP_CONCAT_UNQUOTED.

**Pull request merged**